scim2/scim2_connector.bal

```diff
@@ -146,9 +145,8 @@ function ScimConnector::getListOfUsers() returns (User[]|error) {
                             return userList;
                         }
                     }
-                    error e => {
-                        Error = { message: failedMessage + e.message, cause: e.cause };
-                        return Error;
+                    error err => {
+                        return err;
```

Comment:
here also you can remove `{}` in the match pattern and move return statement to the same line (but you have to move this pattern to the top of the match).